packages/mobile/components/drawers/DrawerManager.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+    import { localize } from '@core/i18n'
+    import { Drawer } from '../../components'
+    import { closeDrawer, DrawerId, drawersStore } from '../../lib/auxiliary/drawer'
+    import ConfirmDrawer from './ConfirmDrawer.svelte'
+    import EnterPasswordDrawer from './EnterPasswordDrawer.svelte'
+    import SelectedActivityDrawer from './SelectedActivityDrawer.svelte'
+
+    const COMPONENTS = {
+        [DrawerId.Confirm]: ConfirmDrawer,
+        [DrawerId.EnterPassword]: EnterPasswordDrawer,
+        [DrawerId.SelectedActivity]: SelectedActivityDrawer,
+    }
+
+    function getStaticTitle(drawerId: DrawerId): string {
+        switch (drawerId) {
+            case DrawerId.EnterPassword:
+                return localize('popups.password.title')
+            case DrawerId.SelectedActivity:
+                return localize('popups.transactionDetails.title')
+            default:
+                return undefined
+        }
+    }
```

Comment:
Hmm what if we put this in the `Drawer` itself and pass the ID. Seems to be consistent we should also create a new locale category for `drawers` similar to popup, but we just make sure to use the same property as the ID so we don't have to write this logic.

Reply:
Maybe we create a constant file for this and add an object for each drawer: 

```
[Drawer.ID.EnterPasswordDrawer]: {
    component: EnterPasswordDrawer,
   title: localize('popups.password.title')
}
```

So we don't have to touch the manager every time we add a drawer

Reply:
Maybe we can create a drawer constant file where we connect the title and component with an id? So we do not have to change the manager every time we add a drawer.

Reply:
I can move the static title to the drawer component, but we cant use the ID to get the static title (it can also be dynamic on multi-step drawers) because in mobile, the modals are also popups, so in the localization file not all strings are in popup, for example wallet switcher, create account, filters...